builders/build/substrate/customizing-chain-specs.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Customizing Chain Specifications
+description: The Chain Specification contains the initial parameters required for launching a new ContainerChain, including the genesis state, token configuration, and more.
+---
+
+# Customizing Chain Specifications {: #customizing-chain-specifications } 
+
+## Introduction {: #introduction }
+
+The chain specification refers to a set of parameters and configurations that define the characteristics and behavior of a new Appchain. It defines the initial settings and state that all nodes participating in the network must agree on to reach consensus and produce blocks. Many of the initial settings can not be changed after the network is launched without generating a completely different chain.
+
+The specification contains two main sections:
+
+- **The client specification** - includes the network parameters, such as the boot nodes the client connects with when joining the network, for example
+- **The genesis state** - includes the initial state upon which all transactions and state transition are executed. It includes the initial registered accounts and balances, and the account with administrator (sudo) privileges, among other information 
+
+This information the chain spec contains can be stored in a rust file (which can be found in the [templates](/builders/build/templates){target=_blank} included in the Tanssi repository) or in a JSON file.
+
+To deploy a ContainerChain through Tanssi, uploading the JSON chain specification file is required. This article will cover the sections and attributes a chain specification contains and how to obtain the file.
+
+## The Client Specification {: #client-specification }
+
+The client specification contains the configuration of the network and other settings (excluding those related to the runtime state): 
+
+- **Name** - name for the specifications
+- **Id** - a unique simple id for the network used to define the storage path in the node
+- **Fork Id** - optional parameter for a network fork identifier
+- **Chain Type** - a parameter that can be set to define the chain type and display additional information or enable additional features (tt can be set to Development, Local, Live, or a custom type)
```

Comment:
```suggestion
- **Chain Type** - a parameter that can be set to define the chain type and display additional information or enable additional features (it can be set to Development, Local, Live, or a custom type)
```